broadcasters/__init__.py

```diff
@@ -31,10 +31,32 @@ def start_outputs(system_queue, demo_output_queue):
         logger.warning("Unable to import modules necessary to run MQTT output.")
         logger.warning("Program will continue to run without this output.")
 
+    try:
```

Comment:
This should be done in a later branch, but we need to meticulously check and make sure the exceptions to this block will prevent the sss from crashing (i.e.. include all possible exceptions)